package.json

```diff
@@ -8,7 +8,7 @@
     "test": "",
     "build": "npm run clean; webpack --env=prod --colors",
     "clean": "rm -rf dist",
-    "dist:server": "pushd dist; python -m SimpleHTTPServer; popd"
+    "dist:server": ". ./scripts/dist_server.sh"
```

Comment:
The leading dot is equivalent to `source`, which I don't think is what we want (it's for injecting variables into our current environment)? If it's executable, then just `./script/dist_server.sh` should do the trick, no?